docs/src/gui/qtvcp.adoc

```diff
@@ -2,100 +2,133 @@
 :toc:
 
 [[cha:qtvcp]]
-= Qtvcp
+= QtVCP
 
 // Custom lang highlight
 // must come after the doc title, to work around a bug in asciidoc 8.6.6
 :ini: {basebackend@docbook:'':ini}
 :hal: {basebackend@docbook:'':hal}
 :ngc: {basebackend@docbook:'':ngc}
 
-Qtvcp is an infrastructure to display a custom CNC screen or control panel in
-LinuxCNC.
-It displays a UI file built with the QTDesigner screen editor or combines this
-with python programming to create a GUI screen for running a CNC machine.
-Qtvcp is completely customizable - you can add different buttons and status
-LEDs etc. or add python code for even finer grain customization.
+QtVCP is an *infrastructure to build custom CNC screens or control
+panels for LinuxCNC*.
 
-.qtdragon - 3 or 4 Axes Sample
+It displays a _UI file built with Qt Designer_ screen editor and
+combines this with _Python programming_ to create a GUI screen for
+running a CNC machine.
+
+QtVCP is completely _customizable_: you can add different buttons and
+status LEDs etc. or add python code for even finer grain customization.
+
+.QtDragon - 3 or 4 Axis Sample
 image::images/silverdragon.png["QTDragon Router",align="left"]
-.qtdefault - 3 Axis Sample
+
+.QtDefault - 3 Axis Sample
 image::images/qt_cnc.png["QTscreen Mill",align="left"]
-.Qtaxis - Self Adjusting Axis Sample
+
+.QtAxis - Self Adjusting Axis Sample
 image::images/qtaxis.png["QTscreen Qtaxis",align="left"]
-.Blender - 4 Axes Sample
+
+.Blender - 4 Axis Sample
```

Comment:
"4 Axes" was correct as 'axes' is the plural of 'axis'.
Was a bit inconsistent, before.

Reply:
​Absolutely ! My bad. That error is seen so often that tends to stick at times :/​